apps/playground/src/app/modules/components/badge-module/badge-module.component.ts

```diff
@@ -2,27 +2,69 @@
  * Angular imports
  */
 import { Component, OnInit } from '@angular/core';
-import { FormBuilder, FormGroup } from '@angular/forms';
+import { FormControl, FormGroup, NonNullableFormBuilder, Validators } from '@angular/forms';
+
+/**
+ * 3rd-party imports
+ */
+import { ElementSize } from '@betagouv/ngx-dsfr';
 import { ThemeColor } from '@betagouv/ngx-dsfr';
 
```

Comment:
Again, we're gonna have some problem with Typed Forms in the Template.
I suggest you do the same thing I showed you for the checkbox MR.